support/src/main/java/se/inera/intyg/common/support/facade/util/ValueToolkit.java

```diff
@@ -121,6 +137,19 @@ public static LocalDate dateValue(Map<String, CertificateDataElement> data, Stri
         return dateValue.getDate();
     }
 
+    public static LocalDate dateValue(Map<String, CertificateDataElement> data, String questionId, String valueId) {
+        final var dataValue = getValue(data, questionId);
+        if (!(dataValue instanceof CertificateDataValueDate)) {
```

Comment:
will this trigger if datevalue == null?

Reply:
Yes. If dateValue is null it won't be an instanceof CertificateDataValueDate and the whole statement will be true.